cfgov/ask_cfpb/models/answer_page.py

```diff
@@ -416,13 +398,7 @@ def clean_search_tags(self):
 
     @property
     def status_string(self):
-        if self.redirect_to_page:
-            if not self.live:
-                return "redirected but not live"
-            else:
-                return "redirected"
-        else:
-            return super().status_string
+        return super().status_string
```

Comment:
Since this is now just returning the parent class `status_string`, you can delete this whole property.